src/components/MovieForm/MovieForm.jsx

```diff
@@ -0,0 +1,255 @@
+                ) : getError ? (
+                    <Alert variant="danger">Something went wrong while trying to get movie data.</Alert>
+                ) : (
+                    <Form noValidate validated={validated} onSubmit={handleSubmit}>
+                        <Form.Group controlId="name">
+                            <Form.Label>Movie title</Form.Label>
+                            <Form.Control
+                                value={data.name}
+                                onChange={(e) => setData({...data, name: e.target.value})}
+                                name="name"
+                                required
+                                placeholder="Interstellar"
+                                type="text"
+                            />
+                            <Form.Control.Feedback type="invalid">
+                                Please provide movie title
+                            </Form.Control.Feedback>
+                        </Form.Group>
+                        <Form.Group controlId="director">
+                            <Form.Label>Director</Form.Label>
+                            <Form.Control
+                                value={data.director}
+                                onChange={(e) => setData({...data, director: e.target.value})}
+                                name="director"
+                                required
+                                placeholder="Christopher Nolan"
+                                type="text"
+                            />
+                            <Form.Control.Feedback type="invalid">
+                                Please provide movie director
+                            </Form.Control.Feedback>
+                        </Form.Group>
+                        <Form.Group controlId="genre">
+                            <Form.Label>Genre</Form.Label>
+                            <Form.Control
+                                value={data.genre}
+                                onChange={(e) => setData({...data, genre: e.target.value})}
+                                name="genre"
+                                required
+                                placeholder="Sci-Fi"
+                                type="text"
+                            />
+                            <Form.Control.Feedback type="invalid">
+                                Please provide genre
+                            </Form.Control.Feedback>
+                        </Form.Group>
+                        <Form.Group controlId="releaseDate">
+                            <Form.Label>Year of release</Form.Label>
+                            <DatePicker
+                                className="form-control"
+                                placeholderText="2014"
+                                dateFormat="yyyy"
+                                selected={data.releaseDate}
+                                onChange={(e) => setData({...data, releaseDate: e})}
```

Comment:
Tu nie ma "e.target.value" ale nie wiem czy jest to potrzebne przy dataPickerze

Reply:
pls wrzuć wszystko na raz a nie po jednym komentarzu xd

Reply:
przy datepickerze jest samo e jako wartość, tak jest skonstrukowany npm